core/trino-main/src/main/java/io/trino/server/security/oauth2/OAuth2ServiceModule.java

```diff
@@ -49,7 +49,17 @@ protected void setup(Binder binder)
                 .setDefault()
                 .to(ScribeJavaOAuth2Client.class)
                 .in(Scopes.SINGLETON);
-        httpClientBinder(binder).bindHttpClient("oauth2-jwk", ForOAuth2.class);
+        httpClientBinder(binder)
+                .bindHttpClient("oauth2-jwk", ForOAuth2.class)
+                // Reset to defaults to override InternalCommunicationModule changes to this client default configuration.
```

Comment:
Please add a comment where defaults for default http client are set.

Reply:
Done